src/meinberlin/meinberlin/static/js/Packages/Proposal/Detail.html

```diff
@@ -1,14 +1,16 @@
 <article class="mein-berlin-proposal-detail">
     <header class="mein-berlin-proposal-detail-header">
+
         <span
-                data-ng-if="data.assignments.length > 0"
+                data-ng-if="data.assignments.length > 0 && isKiezkasse"
```

Comment:
But still: If there are badges, they should be displayed. If the badge names are not used outside of kiezkassen they do no harm.